ui/hooks/bridge/useBridging.test.ts

```diff
@@ -42,11 +42,11 @@ describe('useBridging', () => {
     });
```

Comment:
Can you also add logic in useBridging to link to the portfolio instead of opening the native experience if the wallet's network is not an active src in LD?